src/Surfnet/Stepup/Configuration/InstitutionConfiguration.php

```diff
@@ -110,14 +123,23 @@ public function rebuild()
             throw new DomainException('Cannot rebuild InstitutionConfiguration as it has not been destroyed');
         }
 
+        $useRaLocationsOption            = UseRaLocationsOption::getDefault();
+        $showRaaContactInformationOption = ShowRaaContactInformationOption::getDefault();
+        $allowedSecondFactorList         = AllowedSecondFactorList::blank();
```

Comment:
I mirrored it from the `::create` method for consistency. I think I can inline them in both.